pages/bare_metal_cloud/dedicated_servers/move-failover-ip/guide.en-gb.md

```diff
@@ -49,7 +53,11 @@ With this technology, you can switch IP addresses from one solution to another i
 > If an IP block is moved/added to the vRack, it is no longer linked to a physical server. In this case, any virtual MAC address will be lost during the transfer.
 >
 
-### Moving an IP from the OVHcloud Control Panel
+### Geolocalised IP blocks
+
+If you order an additional IP block on a server but choose a different location (geolocation) for the IP block, this IP block cannot be moved to another server located in the same country as this block. For example, an additional IP block geolocated in Poland (eu-central-war) and ordered on a server located in a French datacentre (eu-west-gra) cannot be moved to a server located in a Polish datacentre (eu-central-war). The IP block can only be moved to an eligible server located in a French datacentre.
```

Comment:
So it's possible to order a block located in Poland on French servers? But, like any block,  it can't be moved between regions (except for GRA/SBG/RBX)?
If I understand correctly, could we add a sentence stating something like this (doesn't sound great so feel free to rephrase): "The geolocation of the IP block is independent of the region to which the block belongs"?

Reply:
Hello @sbraz yes to your first question. Regarding the sentence I am not sure to understand it properly :/

Reply:
Hey @Jessica41 I'll PM you on Monday to discuss this. The idea was to say that even if an IP has geolocation info in a region (e.g. Poland), it doesn't mean that it's usable for servers in this region. It will always be linked to the region for which it was ordered (which is different from the geolocation)?